### PR TITLE
Search bar work cont.

### PR DESCRIPTION
**Pull request by chasulin**

### CODE REVIEW

File: src/components/Search/SearchAutocomplete.tsx [hunk 1]
```diff
@@ -53,20 +53,29 @@ const SearchAutocomplete: React.FC<{
...
-          placeholder="Search for your state, county, or zip"
+          placeholder="Search for a state, county, or zip code"
         />
       )}
       renderOption={option => {
         return <MenuItem region={option} zipCodeInput={zipCodeInput} />;
       }}
+      openOnFocus
+      onOpen={() => {
+        if (setHideMapToggle) setHideMapToggle(true);
```
Comment (ghop02) [inline src/components/Search/SearchAutocomplete.tsx:73]: this is very much a tiny nit, but if statements without brackets make me a bit uncomfy.  Do you have a preference for them? if not would advocate for brackets but if we don't have it set in our eslint, I'm not too worried either

File: src/components/Search/SearchAutocomplete.tsx [hunk 2]
```diff
@@ -12,27 +13,26 @@ function getOptionSelected(option: Region, selectedOption: Region) {
...
     } else setNoOptionsCopy('No location found');
   };
 
   const stringifyOption = (option: Region) => {
-    if (isZip) {
-      if ((option as County).zipCodes) {
-        return `${(option as County).zipCodes.join(' ')}`;
-      }
-      return option.name;
+    if (checkForZipcodeMatch && (option as County).zipCodes) {
```
Comment (ghop02) [inline src/components/Search/SearchAutocomplete.tsx:34]: does this function the same as `(option instanceof County)`? I'm not sure I know the as syntax well enough to know how it behaves if it's not a county.

File: src/components/Search/SearchAutocomplete.tsx [hunk 3]
```diff
@@ -53,20 +53,29 @@ const SearchAutocomplete: React.FC<{
       onChange={onSelect}
       getOptionSelected={getOptionSelected}
       filterOptions={createFilterOptions({
-        matchFrom: isZip ? 'any' : 'start',
+        matchFrom: checkForZipcodeMatch ? 'any' : 'start',
         limit: filterLimit,
         stringify: stringifyOption,
       })}
+      popupIcon={<span />} // adding an empty span removes default MUI arrow icon
```
Comment (ghop02) [inline src/components/Search/SearchAutocomplete.tsx:60]: nice!

**Review by ghop02 — APPROVED** [review]
This looks good to me! I don't have a ton of insight into the style changes but the other code looks good.

File: src/components/Search/SearchAutocomplete.tsx (hunk 1 above)
Comment (chasulin) [reply src/components/Search/SearchAutocomplete.tsx]: no preference at all! just added them in

**Pull request merged**